package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "CaelumWeather",
+  "version": "1.0.0",
+  "description": "World weather application",
+  "main": "app.module.js",
+  "private": true,
+  "scripts": {
+    "dev": "webpack --mode development --config ./config/webpack.config.js",
+    "build": "webpack --mode production --config ./config/webpack.config.js"
```

Comment:
You should make use of HtmlWebpackPlugin in order to produce the index.html to serve your bundle